codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithyjava/generator/library/ToDafnyLibrary.java

```diff
@@ -177,7 +177,11 @@ MethodSpec runtimeException() {
         .endControlFlow()
     );
     return method
-      .addStatement("return $T.create_Opaque($L)", dafnyError, VAR_INPUT)
+      .addStatement(
+        "return $T.create_Opaque($L, dafny.DafnySequence.asString(\"\"))",
```

Comment:
Blocking: I think... there should be some effort here. 
This is the ToDafny side, so this is an instance of `Exception`, or maybe `Throwable`,
but it will have `$L.getMessage`. Technically, the `message` can be null.
Hmm... We would have to do something like ternary:
```
Objects.nonNull($L.getMessage()) ?  dafny.DafnySequence.asString($L.getMessage()) : dafny.DafnySequence.asString(\"\");
```

If you look at the ToDafny code generator,
you should see examples of generating this ternary where ever there is an Option.